confuse/util.py

```diff
@@ -155,7 +161,7 @@ def config_dirs():
     paths = []
 
     if platform.system() == 'Darwin':
-        paths.append(UNIX_DIR_FALLBACK)
+        paths.append(xdg_config_home())
```

Comment:
XDG does not apply to Darwin. These files should end up somewhere in `~/Library` on that platform.